Client/Clique/Common/Extensions/UIViewController+Extension.swift

```diff
@@ -10,38 +10,30 @@ import NVActivityIndicatorView
 
 extension UIViewController {
     
-    func sendOkAlert(
-        with title: String = "",
-        message: String
+    func showAlert(
+            title: String = "",
+            message: String,
+            actions: [UIAlertAction],
+            animated: Bool = true
     ) {
-        let alertController = UIAlertController(title: title,
-                                                message: message ,
-                                                preferredStyle: .alert)
-        let okAction = UIAlertAction(title: Constants.Alerts.defaultOKActionTitle,
-                                     style: .cancel)
-        alertController.addAction(okAction)
-        present(alertController, animated: true)
-    }
-    
-    func sendOKCancelAlert(
-        with title: String = "",
-        message: String,
-        actions: [UIAlertAction]
-    ) {
-        let alertController = UIAlertController(title: title,
-                                                message: message,
-                                                preferredStyle: .alert)
-        actions.forEach { alertController.addAction($0) }
-        present(alertController, animated: true)
+            let alertController = UIAlertController(
+                title: title,
+                message: message,
+                preferredStyle: .alert
+            )
+
+            actions.forEach { alertController.addAction($0) }
+            present(alertController, animated: animated)
     }
-    func sendAlert(
-        with title: String = "",
-        message: String,
-        action: UIAlertAction
+
+    func showOKAlert(
+            title: String = "",
```

Comment:
This can be `nil` as well, so...

```suggestion
            title: String? = nil,
```